x-pack/plugins/security_solution/docs/testing/test_plans/detection_response/rule_management/coverage_overview_dashboard.md

```diff
@@ -0,0 +1,191 @@
+
+### Terminology
+
+- **MITRE ATT&CK**: The [3rd party framework](https://attack.mitre.org/) the dashboard is built upon. It is a knowledge base of attack tactics and techniques adversaries use in real world applications.
+
+- **Tactic**: A generalized category or process that advesaries use to attack a system. Envelops many relevant Mitre Att&ck techniques
+
+- **Technique**: A specific technique advesaries use to attack a system. Can belong to one or more different Mitre Tactics and can potentially contain one or more sub-techniques further describing the process.
+
+- **Rule Activity**: The filter type defining rule status, current options are `enabled` and `disabled`.
+
+- **Rule Source**: The filter type defining rule type, current options are `prebuilt`(from elastic prebuilt rules package) and `custom`(created by user)
+
+-**Initial filter state**: The filters present on initial page load. Rule activity will be set to `enabled`, rule source will be set to `prebuilt` and `custom`.
+
+### Assumptions
+
+- Currently all scenarios below only apply to rules that have correctly mapped `threat` fields (unmapped fields or `threat` fields that don't contain current versioned Mitre Att&ck data will not be displayed in the dashboard)
+- The feature is available under the Basic license
+- "Rules" will be referring to Security rules only (unless stated otherwise)
+- Page always loads with initial filter state
+
+### Non-functional requirements
+
+- Number of rules needs to be under 10k due to [this current bug](https://github.com/elastic/kibana/issues/160698)
+
+## Scenarios
+
+### Coverage overview workflow: base cases
+
+#### **Scenario: No rules installed**
+
+**Automation**: 1 e2e test + 1 unit test 1 integration test.
+
+```Gherkin
+Given no rules installed/created
+When user navigates to Coverage Overview page
+Then an empty grid of all Mitre tactics and techniques are displayed
+```
+
+#### **Scenario: Rules installed**
+
+**Automation**: 1 e2e test + 1 integration test
+
+```Gherkin
+Given rules installed/created
+And rules enabled
+When user navigates to Coverage Overview page
+Then page should render all rule data in grid
+And color tiles according to filters and dashboard legend
+```
+
+#### **Scenario: User clicks on tile**
+
+**Automation**: 1 e2e test + 1 unit test.
+
+```Gherkin
+Given rules installed/created
+And rules enabled
+When user navigates to Coverage Overview page
+And clicks on technique tile with X rules
+Then the popover should display X rule names under their corresponding rule activity secition
+And each name should link to its own rule details page
+```
+
+### Coverage overview workflow: filters
+
+#### **Scenario: No filters are present**
+
+**Automation**: 1 integration test.
+
+```Gherkin
+Given coverage overview page is loaded with rule data
+When no filters or search term is present
+Then all rule data is fetched and rendered
```

Comment:
Maybe expand it to something like `Then the dashboard is rendered according to the rule data`?